src/MoBi.Presentation/Presenter/ObjectPathCreatorAtProcessBuilder.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MoBi.Core.Domain.Model;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.Presentation.Presenter
+{
+   public abstract class ObjectPathCreatorAtProcessBuilder<TBuilder> : ObjectPathCreatorBase where TBuilder : ProcessBuilder
+   {
+      public TBuilder ProcessBuilder { protected get; set; }
+
+      protected ObjectPathCreatorAtProcessBuilder(IObjectPathFactory objectPathFactory, IAliasCreator aliasCreator, IMoBiContext context) : base(objectPathFactory, aliasCreator, context)
+      {
+      }
+
+      protected FormulaUsablePath CreatePathToProcessProperty(IParameter parameter)
+      {
+         var parentProcess = GetProcessBuilderFor(parameter);
+
+         // Always Absolute paths
+         if (parameter.BuildMode != ParameterBuildMode.Local)
+            return CreateFormulaUsablePathFrom(new[] { parentProcess.Name, parameter.Name }, parameter);
+
+         // Parameter is child of process
+         if (parentProcess.Equals(ProcessBuilder))
+            return CreateFormulaUsablePathFrom(new[] { parameter.Name }, parameter);
```

Comment:
Is this correct? Shouldn't the path be `..|parameter.Name` and not just `parameter.Name`?

Reply:
For ProcessRateParameter the ..| will be added when the parameter is created.

This just makes the alias path a relative path instead of absolute (from the builder root).

If the path is not relative, then when ProcessRate is created the ..| will not be added.

In core the service is called `ProcessRateParameterCreator`

Reply:
and the function is called only for `ProcessRate `parameters? Or also for another transport/reaction parameters?

Reply:
`ProcessRateParameterCreator` is used for `ProcessRate` parameters. Is that what you mean?

Reply:
I meant, is the function here `protected FormulaUsablePath CreatePathToProcessProperty(IParameter parameter)` used only for `ProcessRate` parameters?